Src/VimWpf/Implementation/BlockCaret/BlockCaret.cs

```diff
@@ -157,20 +172,22 @@ private bool IsRealCaretVisible
             _textView.LayoutChanged += OnCaretEvent;
             _textView.GotAggregateFocus += OnCaretEvent;
             _textView.LostAggregateFocus += OnCaretEvent;
-            _textView.Caret.PositionChanged += OnCaretPositionChanged;
+            _textView.Selection.SelectionChanged += OnCaretPositionChanged;

```

Comment:
Don't think the method name is correct anymore given the event it's subscribing to.

Reply:
Although it seems odd, this naming does make a certain kind of sense. The selection changed event fires when **any** caret position changes, not just the primary caret. It might be clearer if it was called `OnAnyCaretPositionChanged` or `OnCaretPositionEventOccurred` or even `OnCaretBlinkTimerNeedsUpdating`.

Reply:
The need for this change is so that when you add a new caret, the blink timer is reset and the new caret is visible immediately vs. partway through a blink cycle.